compose/reference/envvars.md

```diff
@@ -4,139 +4,165 @@ keywords: fig, composition, compose, docker, orchestration, cli, reference
+Configures the path to the `ca.pem`, `cert.pem`, and `key.pem` files used for TLS verification.
+
+* **Defaults to:** `~/.docker`.
 
 ## COMPOSE\_HTTP\_TIMEOUT
 
-Configures the time (in seconds) a request to the Docker daemon is allowed to hang before Compose considers
-it failed. Defaults to 60 seconds.
+Defines the timeout period (in seconds) for requests to the Docker daemon after which Compose considers the requests have failed.
+
+* **Defaults to:** 60 seconds.
 
 ## COMPOSE\_TLS\_VERSION
 
-Configure which TLS version is used for TLS communication with the `docker`
-daemon. Defaults to `TLSv1`.
-Supported values are: `TLSv1`, `TLSv1_1`, `TLSv1_2`.
+Defines which TLS version is used for TLS communication with the Docker daemon. 
+
+* **Supported values:** `TLSv1`, `TLSv1_1`, `TLSv1_2`.
+* **Defaults to:** `TLSv1`.
 
 ## COMPOSE\_CONVERT\_WINDOWS\_PATHS
 
-Enable path conversion from Windows-style to Unix-style in volume definitions.
-Users of Docker Machine on Windows should always set this. Defaults to `0`.
-Supported values: `true` or `1` to enable, `false` or `0` to disable.
+When enabled, Compose performs path conversion from Windows-style to Unix-style in volume definitions.
+
+* **Supported values:** 
+    * `true` or `1`, to enable, 
+    * `false` or `0`, to disable.
+
+* **Defaults to:** `0`
 
 ## COMPOSE\_PATH\_SEPARATOR
 
-If set, the value of the `COMPOSE_FILE` environment variable is separated
-using this character as path separator.
+Specifies a different path separator for items listed in `COMPOSE_FILE`. 
+
+Default path separator in Linux and macOS the path separator is `:`, on Windows it is `;`.
 
 ## COMPOSE\_FORCE\_WINDOWS\_HOST
 
-If set, volume declarations using the [short syntax](../compose-file/compose-file-v3.md#short-syntax-3)
-are parsed assuming the host path is a Windows path, even if Compose is
+When enabled, volume declarations using the [short syntax](../compose-file/compose-file-v3.md#short-syntax-3) are parsed assuming the host path is a Windows path, even if Compose is
 running on a UNIX-based system.
-Supported values: `true` or `1` to enable, `false` or `0` to disable.
+
+* **Supported values:** 
+    * `true` or `1`, to enable, 
+    * `false` or `0`, to disable.
 
 ## COMPOSE\_IGNORE\_ORPHANS
 
-If set, Compose doesn't try to detect orphaned containers for the project.
-Supported values: `true` or `1` to enable, `false` or `0` to disable.
+When enabled Compose doesn't try to detect orphaned containers for the project.
+
+* **Supported values:** 
+    * `true` or `1`, to enable, 
+    * `false` or `0`, to disable.
 
 ## COMPOSE\_PARALLEL\_LIMIT
 
-Sets a limit for the number of operations Compose can execute in parallel. The
-default value is `64`, and may not be set lower than `2`.
+Sets a limit for the number of operations Compose can execute in parallel.
+
+* **Supported values:** minimum value is `2`.
+* **Defaults to:** `64`.
```

Comment:
This variable is not implemented in V2

Reply:
(Assumed COMPOSE_PARALLEL_LIMIT) Added deprecation notice.